config/registry/aws/modules/cloudfront-distribution.yaml

```diff
@@ -19,18 +19,21 @@ inputs:
     description: Name of the access log bucket
     default: null
   - name: bucket_name
-    user_facing: false
+    user_facing: true
+    validator: str(required=False)
     description: The name of the s3 bucket to link to this cloudfront distribution
     default: ""
   - name: origin_access_identity_path
-    user_facing: false
+    user_facing: true
+    validator: str(required=False)
     description: The Cloudfront OAI path to use to access the buckets
     default: ""
   - name: default_page_file
     user_facing: true
     validator: str(required=False)
     description: The name of the existing s3 object in your bucket which will serve as the default page.
-    default: null
+    default: "index.html"
+    force_update_default_counter: 1
```

Comment:
Do we need to bump this?

Reply:
I did because it was not changing the default on my example because I already ran it with default null

Reply:
Let's delete this? We should not commit testing code.